server/repository/file_pairs.go

```diff
@@ -0,0 +1,47 @@
+package repository
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/src-d/code-annotation/server/model"
+)
+
+// FilePairs repository
+type FilePairs struct {
+	db *sql.DB
+}
+
+// NewFilePairs returns a new FilePairs repository
+func NewFilePairs(db *sql.DB) *FilePairs {
+	return &FilePairs{db: db}
+}
+
+// getWithQuery builds a FilePair from the given sql QueryRow. If the FilePair
+// does not exist, it returns nil, nil
+func (repo *FilePairs) getWithQuery(queryRow *sql.Row) (*model.FilePair, error) {
+	var pair model.FilePair
+
+	var a, b model.File
```

Comment:
something weird is going on here. You never use `a`&`b` beside scanning values to them.

Reply:
Good catch, this should be pair.Left and Right. I missed it because the serializer does nothing with these two values.